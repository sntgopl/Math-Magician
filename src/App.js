import Calculator from "./Components/Calculator";
import React from "react";
import './app.css';

function App() {
  return (
    <div className="application">
      <Calculator/>
    </div>
  );
}

export default App;