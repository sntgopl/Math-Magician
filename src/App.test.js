import React from 'react';
import renderer from 'react-test-renderer';
import { Home, About, Quotes } from './App';

it('renders correctly', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
  const tree = renderer
    .create(<About />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
  const tree = renderer
    .create(<Quotes />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
