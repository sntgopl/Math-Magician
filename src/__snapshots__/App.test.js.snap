// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="HomePage"
>
  <main
    className="HomeInfo"
  >
    <h2
      className="HomeTitle"
    >
      Welcome to the homepage!
    </h2>
    <p
      className="HomeText"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam hendrerit quam quis lorem luctus suscipit quis et eros. Suspendisse potenti. Cras mollis condimentum placerat. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Cras velit ipsum, ultricies at est sit amet, aliquam suscipit felis. Donec id interdum odio. Duis sed enim vel enim faucibus aliquet. Donec aliquet lectus et urna egestas luctus. Maecenas eget lectus iaculis, auctor dui vel, vulputate lectus. Sed semper iaculis bibendum. In nunc arcu, vestibulum sed interdum eget, dignissim vulputate metus. Cras pulvinar quam sit amet arcu ullamcorper, non porta magna laoreet. Aliquam enim augue, lobortis at mauris eget, pharetra bibendum enim. Nam sollicitudin lorem sed ullamcorper imperdiet. Mauris id sapien id tellus consectetur blandit vel eu eros. Cras tempus elementum tortor tempor tincidunt. Suspendisse varius, massa ut sagittis bibendum, lacus ligula accumsan tortor, sed placerat massa purus vel orci. Ut bibendum porta arcu at dignissim. Phasellus ullamcorper metus vitae diam placerat dictum. Etiam ut velit eu enim aliquet lobortis. Sed feugiat pharetra vestibulum. Etiam vitae luctus ex. Donec ut magna elementum, convallis ex sed, sollicitudin nulla. Nulla a porttitor erat. Quisque pulvinar, sem non elementum congue, leo lorem fermentum lacus, ut volutpat sem magna id mi. Sed elit tortor, pretium eget sapien sit amet, suscipit maximus arcu. Integer pretium, mauris sit amet varius eleifend, mauris dui euismod neque, vel ultricies purus justo eget turpis. Proin vel ante vel risus maximus ullamcorper. Cras luctus, purus sed rhoncus bibendum, ex nisi bibendum odio, eu blandit metus dui non eros. Integer non magna dui. Praesent blandit nunc libero, efficitur ultricies augue sagittis nec. Etiam blandit urna ut rhoncus tincidunt. Quisque blandit feugiat lacus, in fermentum metus convallis vel. Fusce lectus tortor, efficitur ut pretium et, cursus in sem. Curabitur pulvinar ac mauris a tempor. Quisque at elementum odio, tempus consectetur augue.
    </p>
  </main>
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="AboutPage"
>
  <main
    className="AboutInfo"
  >
    <h2
      className="AboutTitle"
    >
      Let\`s do some Math!
    </h2>
    <div
      className="Calculator"
    >
      <p
        className="result"
        data-testid="display"
      >
         
        0
      </p>
      <div
        className="board"
      >
        <button
          data-testid="AC"
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          %
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          รท
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          7
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          8
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          9
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          x
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          4
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          5
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          6
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          -
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          1
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          2
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          3
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          +
        </button>
        <button
          className="btn0"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          .
        </button>
        <button
          className="orange"
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </main>
</div>
`;

exports[`renders correctly 3`] = `
<div
  className="QuotesPage"
>
  <main
    className="QuotesInfo"
  >
    <h2
      className="QuotesTitle"
    >
      Daily Quote
    </h2>
    <p
      className="Quote"
    >
      If you can\`t explain it simply, you don\`t understand it well enough. - Albert Einstein
    </p>
  </main>
</div>
`;
